server/auth/github/client.go

```diff
@@ -173,6 +177,29 @@ func (c *Client) IsMember(organization, token string) (bool, error) {
 	return true, nil
 }
 
+// IsTeamMember returns true if the given user is a member of the team.
+func (c *Client) IsTeamMember(teamId, name string, token string) (bool, error) {
```

Comment:
Problem with that right now is backwards compat, since existing JWT tokens won't have the `GitHubLogin` field embedded. It wouldn't be a big deal to require Empire installations to re-generate a new signing secret and have users log in again, but if we can avoid that it's probably best.

I think I'd prefer just hitting the /user endpoint of the github api to grab their login using the given token, but either one is fine imo. Up to you.